cg/services/orders/order_service/utils.py

```diff
@@ -24,6 +24,9 @@ def create_orders_response(
 def _add_summaries(orders: list[Order], summaries: list[OrderSummary]) -> list[Order]:
     order_map = {order.id: order for order in orders}
     for summary in summaries:
-        order = order_map[summary.order_id]
-        order.summary = summary
+        try:
+            order = order_map[summary.order_id]
+            order.summary = summary
+        except KeyError:
+            continue
```

Comment:
Which situation do the exceptions solve? Is there a missmatch between the number of orders and summaries?

Reply:
We can leave them out, but this one occurs if an analysis is tagged with an order that has been removed.

Reply:
Good point, this one can be skipped. There will always be a matching summary and order

Reply:
This exception was hiding a bug from a function (`filter_orders_by_ids`) :+1: